docs/.link-check-errors.txt

```diff
@@ -1,2 +1,3 @@
 Broken links found by `pnpm docs:link-check` that exist:
 
+- https://mui.com/material-ui/react-button/#loading-button
```

Comment:
Added by `pnpm docs:link-check`. I am not sure what we should do here.

Reply:
The "Loading button" heading has been renamed to "Loading" (see https://github.com/mui/material-ui/pull/42987/files#diff-fcd78cd26d8ab85cd546c9e0ac0ef385fea26d78b33bd32e00407c470d0f2c42R122), so the markdown files linking to `https://mui.com/material-ui/react-button/#loading-button` should be updated and link to `https://mui.com/material-ui/react-button/#loading` instead (note the different hash).

Reply:
We can link to v5 I guess. We're already doing that in that particular page: https://github.com/mui/material-ui/blob/v6.1.6/docs/pages/blog/mui-core-v5.md?plain=1#L239